src/pages/index.js

```diff
@@ -0,0 +1,333 @@
+  headers: {
+    authorization: "166c6acc-ad57-4210-878a-b822a32c3faa",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([userInfo, cards]) => {
+    cards.forEach((item) => {
+      renderCard(item, "append");
+    });
+
+    profileAvatar.src = userInfo.avatar;
+    profileName.textContent = userInfo.name;
+    jobElement.textContent = userInfo.about;
+  })
+  .catch(console.error);
+
+// Profile elements
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const cardModalButton = document.querySelector(".profile__add-btn");
+const avatarModalButton = document.querySelector(".profile__avatar-btn");
+const profileName = document.querySelector(".profile__name");
+const jobElement = document.querySelector(".profile__description");
+const profileAvatar = document.querySelector(".profile__avatar");
+
+// Edit form elements
+const editModal = document.querySelector("#edit-modal");
+const editFormElement = editModal.querySelector(".modal__form");
+const editModalCloseBtn = editModal.querySelector(".modal__close-btn");
+const editModalNameInput = editModal.querySelector("#profile-name-input");
+const editModalDescriptionInput = editModal.querySelector(
+  "#profile-description-input"
+);
+
+// Delete form elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteModalCloseBtn = deleteModal.querySelector(".modal__close-btn");
+const deleteForm = deleteModal.querySelector(".modal__form");
+const cancelButton = document.querySelector(".modal__cancel-btn");
+
+// Card form elements
+const cardModal = document.querySelector("#add-card-modal");
+const cardFormElement = cardModal.querySelector(".modal__form");
+const cardSubmitBtn = cardModal.querySelector(".modal__submit-btn");
+const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+
+// Preview image popup elements
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+const modalCloseTypePreview = previewModal.querySelector(
+  ".modal__close-btn_type_preview"
+);
+
+// Card related elements
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+const modals = document.querySelectorAll(".modal");
+
+let selectedCard, selectedCardId;
+
+// Avatar form element
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarFormElement = avatarModal.querySelector(".modal__form");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+
+modalCloseTypePreview.addEventListener("click", () => {
```

Comment:
Please, delete the redundant `close icon` listeners because you handle them in line 300 **universally**